include/swift/IDE/Formatting.h

```diff
@@ -17,7 +17,9 @@ namespace swift {
 namespace ide {
 
 struct CodeFormatOptions {
+public:
   bool UseTabs = false;
```

Comment:
I removed individual counterparts in `swift-format`'s source file because that approach isn't going to be scalable as we and more and more format options.